### ISSUE TITLE
[BUG] Server not loading vanilla PBOs (Warning Message: You cannot play/edit this mission; it is dependent on downloadable content that has been deleted.\na3_characters_f)

### ISSUE THREAD

**Issue by Drift91** [Foxlider/FASTER#72]
**Describe the bug**
After launching my server via FASTER I get the following RPT log when I attempt to load into the game and the server crashes:
```
Warning Message: You cannot play/edit this mission; it is dependent on downloadable content that has been deleted.\na3_characters_f
```
This basically tells me that for some reason the server is being told by FASTER not to load the vanilla PBOs from the `addons` folder.

**To Reproduce**
Steps to reproduce the behaviour:
1. Simply install, set up and launch a server and attempt to play on it.

**Expected behaviour**
The mission to load without issue.

**Actual behaviour**
The server crashes due to missing dependencies that aren't being loaded despite being present in the vanilla `addons` folder.

**Screenshots**
N/A

**OS Version (required):**
 - OS: Windows Server 2022 x64
 - Version 1.8c

**Additional context**
I'm unsure if the issue maybe has something to do with the fact that I set FASTER to deploy mods to the vanilla install directory.

**Comment by Drift91** [comment]
After redownloading 1.7c and clicking "Update" FASTER decided to redownload all the vanilla PBOs. Because of this I'm guessing 1.8c was botching the download, unless SteamCMD just didn't like that the manifest was outdated or something. I had clicked the update button numerous times on 1.8c and it didn't see anything wrong with the files.

**Comment by MildlyInterested** [comment]
Thats a default error that is supposed to be there. If your mission isn't loading its because of a different issue.

**Comment by Drift91** [comment]
Where is this mentioned in the Discord? I'd like to see an explanation of why it's supposed to be there, as I feel it's rather idiotic to deliberately put an inaccurate error message that will mislead server owners.

Anyway, going to close this issue since you are right about it being caused by another issue. I later realized that the addons were in fact being loaded as they showed in the RPT file as being active. Turns out Windows failed to copy my new mod preset over remote desktop to the server box and when I loaded it it was missing the map the mission is on. Kind of silly that the RPT showed a vanilla class instead of the actual missing mod.
